frontend/packages/app/src/clutch.config.js

```diff
@@ -46,4 +46,41 @@ module.exports = {
       },
     },
   },
+  "@clutch-sh/experimentation": {
+    listExperiments: {
+      trending: true,
```

Comment:
We should probably remove this from trending until we finish making changes

Reply:
I forgot to clean it up 👍 updating